packages/react/src/components/Inbox.tsx

```diff
@@ -39,16 +40,25 @@ export const Inbox = React.memo((props: InboxProps) => {
     return <Renderer options={options}>{children}</Renderer>;
   }
 
-  const { renderNotification, renderBell, ...options } = props;
+  const {
+    open,
+    renderNotification,
+    renderBell,
+    onNotificationClick,
+    onPrimaryActionClick,
+    onSecondaryActionClick,
+    ...options
+  } = props;
 
   return (
     <Renderer options={options}>
       <DefaultInbox
+        open={props.open}
         renderNotification={renderNotification}
         renderBell={renderBell}
-        onNotificationClick={props.onNotificationClick}
-        onPrimaryActionClick={props.onPrimaryActionClick}
-        onSecondaryActionClick={props.onSecondaryActionClick}
+        onNotificationClick={onNotificationClick}
+        onPrimaryActionClick={onPrimaryActionClick}
+        onSecondaryActionClick={onSecondaryActionClick}
```

Comment:
It's missing the tabs props

Reply:
The tabs are propagated to the Renderer which then sets it on the NovuUI and later on the InboxProvider

Reply:
Yes, @BiswaViraj , the tabs are a prop on the provider. they won't be part of the `mountInbox` options. Essentially, they're passed when you do `new NovuUI`